documentation/source/NuRadioReco/pages/times.rst

```diff
@@ -25,28 +25,32 @@ Trace start times in E-fields
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Work similar to trace_start_time in channels, but with a caveat: Since EM-waves move, electric_field objects hold a position (relative to the station they are associated with). The E-field is therefore defined as the field an observer would measure at the given position. Note that this position does not necessarily have to coincide with the position of a channel the E-field is associated with. This is the case for (some) cosmic-ray simulations where the same E-field at the surface is used for all surface LPDAs.
 
-overview of modules that alter time
-===================================
+Overview of modules that affect time
+------------------------------------
 We list all relevant modules that is used for a MC simulation and reconstruction. For a pure data reconstruction, the first few modules are just not used
 
-* readCoREAS: CoREAS reader prepends n samples to the simulated trace. This is done so that the trace does not directly start with the pulse and to have a good frequency resolution.
+* `NuRadioReco.modules.io.coreas`: CoREAS reader prepends n samples to the simulated trace. This is done so that the trace does not directly start with the pulse and to have a good frequency resolution.
 
-* efieldToVoltageConverter: the traces are rolled (rotated) according to the time delay due to the geometric separation of the antennas and cable delays.
+* `NuRadioReco.modules.efieldToVoltageConverter`: the traces are rolled (rotated) according to the time delay due to the geometric separation of the antennas and cable delays.
```

Comment:
This isnt this wrong? This module only changes the trace start time but does no rolling.